app.js

```diff
@@ -13,7 +13,7 @@ const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
 app.use(logger(formatsLogger))
 app.use(cors())
 app.use(express.json())
-app.use(express.static("pablik"))
+app.use(express.static("pablic"))
```

Comment:
```suggestion
app.use(express.static("public"))
```назву папки також поправте